### PR TITLE
makes cloning vats announce on radio when a clone is ejected

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

makes cloning vats announce on medical and command radio when a clone is ejected
also fixes autodoc console using wrong radio channel to announce ejections

## Why It's Good For The Game

extra chance of clone soldiers being actually put in squads

## Changelog
:cl:
add: makes cloning vats announce on radio when a clone is ejected
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/cloning/cloning.dm [hunk 1]
```diff
@@ -241,6 +260,7 @@ You remember nothing of your past life.
 
 You are weak, best rest up and get your strength before fighting.</span>"})
 	occupant.vomit()
-
+	linked_console.radio.talk_into(src, "<b>New clone: [occupant] has been grown in [src] at: [get_area(src)].</b>", RADIO_CHANNEL_MEDICAL)
```
Comment (Fikou) [inline code/game/objects/machinery/cloning/cloning.dm:263]: maybe some text like for command would be nice here?

Comment (Hughgent) [reply code/game/objects/machinery/cloning/cloning.dm:263]: Command gets the medical channel.

**Review by Hughgent — APPROVED** [review]
LGTM

File: code/game/objects/machinery/cloning/cloning.dm (hunk 1 above)
Comment (Fikou) [reply code/game/objects/machinery/cloning/cloning.dm:263]: i meant for an objective for medical like how command has an objective to give vat to a squad

File: code/game/objects/machinery/cloning/cloning.dm [hunk 2]
```diff
@@ -111,6 +127,9 @@ These act as a respawn mechanic growning a body and offering it up to ghosts.
 		eject_user()
 
 	QDEL_NULL(beaker)
+	if(linked_console)
+		linked_console.linked_machine = null
+		linked_console = null
```
Comment (TiviPlus) [inline code/game/objects/machinery/cloning/cloning.dm:132]:
```suggestion
	linked_console?.linked_machine = null
	linked_console = null
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**